src/api/app/models/user.rb

```diff
@@ -835,18 +835,18 @@ def involved_reviews
   end
 
   # list requests involving this user
-  def declined_requests
-    BsRequest.collection(user: login, states: %w(declined), roles: %w(creator))
+  def declined_requests(search = nil)
+    BsRequest.collection(user: self.login, states: %w(declined), roles: %w(creator), search: search)
```

Comment:
Guess you should remove the self since @Ana06 just banned using it:-) -> https://github.com/openSUSE/open-build-service/pull/2271